test_opensearchpy/test_async/test_connection.py

```diff
@@ -236,7 +236,7 @@ async def test_nowarn_when_test_uses_https_if_verify_certs_is_off(self) -> None:
                 assert (
                     str(w[0].message) == "enable_cleanup_closed ignored because "
                     "https://github.com/python/cpython/pull/118960 is fixed in "
-                    "Python version sys.version_info(major=3, minor=12, micro=7, releaselevel='final', serial=0)"
+                    "Python version sys.version_info(major=3, minor=12, micro=8, releaselevel='final', serial=0)"
```

Comment:
This fine, but I'd prefer we fixed it once and for all by checking that the string contains `https://github.com/python/cpython/pull/118960` and none of the other stuff.